src/main/java/jdbchomework/TestMain.java

```diff
@@ -0,0 +1,24 @@
+package jdbchomework;
+
+import jdbchomework.utils.HibernateUtil;
+import jdbchomework.utils.InjectorHibTest;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+public class TestMain {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1118)